File: lab07/count_word.pl
#!/usr/bin/perl -w
if ($#ARGV != 0){
    print("Please give one word!\n");
    exit(0);
}
$count = 0;
$word = lc($ARGV[0]);
@txt = <STDIN>;
foreach $l (@txt){
    $l =~ s/'s/ s/g;
    $l =~ s/[^a-zA-Z]/ /g;
    @line = split/\s+/, lc($l);
    foreach $w (@line){
	if ($w =~ /^$word$/){
	    $count++;
	}
    }
}
if ($count > 1) {
    print "$word occurred $count times\n";
}
else {
    print "$word occurred $count time\n";
}

File: lab07/count_word.py
#!/usr/bin/python3
import sys
import re
w_count = 0
for line in sys.stdin:
    line = re.sub(r'[^A-Za-z]',' ',line)
    line = re.sub(r'\'s',' s',line)
    word = re.split(r'\s+',line.lower())
    for x in word:
        if x == sys.argv[1].lower():
           w_count += 1
print("{} occurred {} times".format(sys.argv[1].lower(),w_count)) 
